### PR TITLE
Export outpack_root_open

### PR DESCRIPTION
**Pull request by richfitz**
We need this for the orderly MVP. Still punting on exactly what is in a root object though :)

### CODE REVIEW

File: R/root.R [hunk 1]
```diff
@@ -102,30 +102,43 @@ outpack_root <- R6::R6Class(
...
+##' Open an existing outpack root. This returns a "root" object, which
+##' can be passed through to various outpack functions. The root
+##' object is the same as that returned by
+##' [outpack::outpack_init] and will be documented once the
+##' interface stabilises.
+##'
+##' @title Open outpack root
+##'
+##' @param path The path to look for the root; must be an existing
+##'   directory. If `NULL`
```
Comment (hillalex) [inline R/root.R]: If `NULL` what?

**Review by hillalex — APPROVED** [review]
Looks good

File: R/root.R (hunk 1 above)
Comment (richfitz) [reply R/root.R]: oops

Comment (richfitz) [reply R/root.R]: This is the downside of somewhat documentation development. I started explaining something, decided it was a bad idea, removed the thing I was explaining and forgot to finish documenting

**Pull request merged**